### ISSUE TITLE
Unable to convert large portions of world because brush limit

### ISSUE THREAD

**Issue by stopmotio** [Garten/sourcecraft#12]
Hits the brush limit when converting large areas. Not your fault at all and idk how this is to be solved directly but I only wanted to do this to make the edge not noticable. Maybe a 3d skybox option? Idk man

**Comment by Garten** [comment]
I agree, a 3d skybox would be a nice option. In theory it could be automatically generated from the minecraft world, maybe a translation not block-wise but one that represents a lot of terrain but in low resolution.

**Comment by stopmotio** [comment]
The 3D skybox could just be the surface blocks to help with the brush limit, too, which I think would help with the brush limit issue
